packages/grafana-data/src/utils/useObservable.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useState } from 'react';
+import { Observable } from 'rxjs';
+
+export function useObservable<T>(observable: Observable<T>, initialValue: T): T {
```

Comment:
why @grafana/data

Is this any different from https://streamich.github.io/react-use/?path=/story/state-useobservable--docs

Reply:
no, can remove this one.